e2e-tests/tools/daemon-util/index.js

```diff
@@ -39,6 +42,15 @@ function startNpmScript(script, color) {
 
   child.on('exit', () => {
     console.log(normal(`Finished "npm run ${script}"`));
+    // This is needed to kill the node server process on windows which is not automatically killed
```

Comment:
Since we're using `spawnWith: {shell: true}` in the configuration above, on windows, these spawned shell node processes aren't killed when the parent script exits. 
E.g. If you run `npm test`, it spawns a shell that runs `npm run okta-hosted-login-server`. But when `npm test` ends, the spawned shell is still running which will cause the next test to fail since the server runs on the same port 8080.
Hence we need to kill the spawned node process on windows using `TASKKILL`